AirCasting/MainTabBarView.swift

```diff
@@ -175,10 +197,15 @@ class FinishAndSyncButtonTapped: ObservableObject {
 }
 
 class ReorderButtonTapped: ObservableObject {
-    @Published var reorderIsON = false
+    @Published var reorderIsOn = false
     @Published var isHidden = false
 }
 
+class SearchAndFollowButtonTapped: ObservableObject {
```

Comment:
I don't like those names. It's not a proper class name. Maybe try to come up with something more proper and rename (both this and Reorder)